hydra/jobsets.nix

```diff
@@ -0,0 +1,65 @@
+{ prs }:
```

Comment:
How do we pass in this value?

Reply:
this values comes from `hydra.json`. Just pushed it, you need to change git paths to your repository.